Docker/Alfresco/acs-share/share-config-custom.xml

```diff
@@ -0,0 +1,741 @@
+
+            Example config entry:
+              <share-url repositoryId="622f9533-2a1e-48fe-af4e-ee9e41667ea4">http://new-york-office:8080/share/</share-url>
+         -->
+      </share-urls>
+   </config>
+
+   <!-- Document Library config section -->
+   <config evaluator="string-compare" condition="DocumentLibrary" replace="true">
+
+      <tree>
+         <!--
+            Whether the folder Tree component should enumerate child folders or not.
+            This is a relatively expensive operation, so should be set to "false" for Repositories with broad folder structures.
+         -->
+         <evaluate-child-folders>false</evaluate-child-folders>
+         
+         <!--
+            Optionally limit the number of folders shown in treeview throughout Share.
+         -->
+         <maximum-folder-count>1000</maximum-folder-count>
+         
+         <!--  
+            Default timeout in milliseconds for folder Tree component to recieve response from Repository
+         -->
+         <timeout>7000</timeout>
+      </tree>
+
+      <!--
+         Used by "Manage Rules" -> "Add aspect" action.
+
+         If an aspect has been specified without a title element in the content model,
+         or you need to support multiple languages,
+         then an i18n file is needed on the Repo AMP/JAR extension side for the aspect to
+         be visible when creating rules:
+
+          custom_customModel.aspect.custom_myaspect.title=My Aspect
+
+         Used by the "Manage Aspects" action.
+
+         For the aspect to have a localised label add relevant i18n string(s) in a Share AMP/JAR extension:
+
+          aspect.custom_myaspect=My Aspect
+      -->
+      <aspects>
+         <!-- Aspects that a user can see -->
+         <visible>
+            <aspect name="cm:generalclassifiable" />
+            <aspect name="cm:complianceable" />
+            <aspect name="cm:dublincore" />
+            <aspect name="cm:effectivity" />
+            <aspect name="cm:summarizable" />
+            <aspect name="cm:versionable" />
+            <aspect name="cm:templatable" />
+            <aspect name="cm:emailed" />
+            <aspect name="emailserver:aliasable" />
+            <aspect name="cm:taggable" />
+            <aspect name="app:inlineeditable" />
+            <aspect name="cm:geographic" />
+            <aspect name="exif:exif" />
+            <aspect name="audio:audio" />
+            <aspect name="cm:indexControl" />
+            <aspect name="dp:restrictable" />
+            <aspect name="smf:customConfigSmartFolder" />
+            <aspect name="smf:systemConfigSmartFolder" />
+         </visible>
+
+         <!-- Aspects that a user can add. Same as "visible" if left empty -->
+         <addable>
+         </addable>
+
+         <!-- Aspects that a user can remove. Same as "visible" if left empty -->
+         <removeable>
+         </removeable>
```

Comment:
If this is the default share-config-custom.xml, with a few changes for our needs, can you add some comments to clearly identify our customizations?  And maybe at the start of the file, add something like "ArkCase customizations identified by such-and-such opening and closing comments".

Reply:
Yes, it is the default share-config-custom.xml with a few modifications. That's a good idea, I will add some comments to document those modifications.